templates/form/formButtons.tpl

```diff
@@ -30,9 +30,9 @@
 					cancelUrl=$FBV_cancelUrl}
 		{/if}
 		{if $FBV_formReset}
-			{fbvElement type="link" class="resetButton" id="resetButton"|concat:"-":uniqid label=$FBV_cancelText}
+			<a href="#" id="resetButton-{$uniqid}" class="resetButton">{translate key=$FBV_cancelText}</a>
```

Comment:
Ok I've chucked that in there but can't actually find a use of the reset button to test with. Maybe it can be removed, along with the related code in `FormHandler.js`?